packages/neon_framework/lib/src/router.dart

```diff
@@ -294,7 +294,7 @@ class LoginCheckServerStatusRoute extends GoRouteData {
     required this.serverUrl,
     required String this.loginName,
     required String this.password,
-  }) : assert(!kIsWeb, 'Might leak the password to the browser history');
+  });
```

Comment:
This is still an issue.
I don't think it should land like this :(

Reply:
I want to make some tests, because I believe the query parameter will not actually be a query parameter and just a part of the segment.

Reply:
I can confirm: The optional parameters are also part of the segment, but formatted like they as if they were query parameters.